doc/enterprise-shield/c_enterprise_shield_use_cases.md

```diff
@@ -0,0 +1,78 @@
+2.  [About Enterprise Shield™](o_enterprise_shield_checklist.md#about-enterprise-shield)
+3.  **Common Enterprise Shield™ Use Cases**
+4.  [Setting Up the Gateway](../about/setup-guide.md)
+5.  [Walkthrough: Configure Enterprise Shield™](p_enterprise_shield_config.md)
+6.  [Walkthrough: Configure Enterprise Shield™ for High Availability](p_enterprise_shield_cluster.md)
+
+The use case configurations described in this topic vary depending on the level of security, such as if the Gateways are clustered for high availability, whether network transports are used, and so on. Also, the sample topologies use two Gateway services instead of just one to demonstrate the Enterprise Shield™ reverse connectivity behavior in a configuration with multiple services.
+
+Each of the Enterprise Shield™ use cases configures Enterprise Shield™ on a single pair of Gateways (DMZ and internal Gateways), and then extends that topology to a cluster configuration. To configure Enterprise Shield™ and control Gateway operation, you configure elements in the Gateway configuration (gateway-config.xml) file. For complete information about Gateway configuration, see [About Gateway Configuration](../admin-reference/c_configure_gateway_concepts.md). Also, the complete Gateway configuration files used for these use cases are available on `kaazing.org`. The configuration file names are included in each use case description that follows.
+
+Use Case 1: Test Drive Enterprise Shield™
+------------------------------------------------------------------
+
+This is the simplest Enterprise Shield™ architecture for the purpose of prototyping and evaluating how this feature can meet your business needs. No security is configured, which reduces the complexity of setting up this architecture. Also, it is easy to demonstrate this configuration on a single host environment, such as a laptop.
+
+![Use Case 1: Test Drive Enterprise Shield™](../images/es_usecase3.png)
+
+**Figure 1: Test Drive Enterprise Shield™**
+
+The following complete configuration files are available on `kaazing.org`:
+
+-   Single pair of Gateways: 
+  -   Internal Gateway: `enterprise-shield-use-case-1-single-internal-gw.xml`
+  -   DMZ Gateway: `enterprise-shield-use-case-1-single-dmz-gw.xml` 
+-   Cluster of Gateways:
+  -   Internal Gateway: `enterprise-shield-use-case-1-cluster-internal-gw.xml`
+  -   DMZ Gateway: `enterprise-shield-use-case-1-cluster-dmz-gw.xml`
+
+Use Case 2: Security Over the Web
+--------------------------------------------------------
+
+This topology demonstrates the architecture for secure communication over the Web that uses TLS between the client and the DMZ Gateway (represented by locks). The communication in the data center is unsecure. A production deployment should always use security for connections over the Web. Use this topology when you want secure Web communication, but network transport security is not a requirement between the DMZ and the trusted network.
```

Comment:
> The communication in the data center is unsecure. A production deployment should always use security for connections over the Web.

Swap these two sentences as "A production deployment should always use security for connections over the Web" is more aligned with the first sentence of the paragraph.

Also consider adding something to the effect of "This use case is the same as use case 1, but with security added between the client and the DMZ gateway." That way the reader understands the delta between use case 1 and 2.